### PR TITLE
docs: added doc for change events

### PR DESCRIPTION
**Pull request by mattayres**

### CODE REVIEW

File: docs/change-events/introduction.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Each operation of a transaction is recorded as a change event. You can stream these change events per collection in
+real-time. The table below describes the fields of the JSON for each change event.
+
+| Field          | Description                                                                                      |
+| -------------- | ------------------------------------------------------------------------------------------------ |
+| **tx_id**      | Unique id shared by all operations of a transaction.                                             |
+| **collection** | Name of the collection the change applies to.                                                    |
+| **op**         | Operation type, one of: `insert`, `replace`, `update`, `updateRange`, `delete` or `deleteRange`. |
+| **key**        | Key for operations of `insert`, `replace`, `update` or `delete`.                                 |
```
Comment (himank) [inline docs/change-events/introduction.md:11]: As key,lkey,rkey are internal to us let's not expose it?

Comment (mattayres) [reply docs/change-events/introduction.md:11]: It seems like the keys would be useful to consumers?

Comment (himank) [reply docs/change-events/introduction.md:11]: Probably not, as it is encoded as well. If needed we can think about adding it later.

Comment (mattayres) [reply docs/change-events/introduction.md:11]: Oh right, it's encoded. I'll remove for now.

Comment (mattayres) [reply docs/change-events/introduction.md]: Fixed.

**Review by himank — APPROVED** [review]

**Pull request merged**